plugins/woocommerce-admin/client/marketplace/components/product-loader/product-loader.tsx

```diff
@@ -5,15 +5,25 @@
 /**
  * Internal dependencies
  */
-import './product-loader.scss';
+import ProductListHeader from '../product-list-header/product-list-header';
+import ProductCard from '../product-card/product-card';
+
+interface ProductLoaderProps {
+	hasTitle?: boolean;
+	placeholderCount?: number;
+}
+
+export default function ProductLoader(props: ProductLoaderProps): JSX.Element {
+	const { hasTitle } = props;
+	const placeholderCount = props.placeholderCount || 12
 
-export default function ProductLoader(): JSX.Element {
 	return (
-		<div className="woocommerce-marketplace__product-loader">
-			<div className="woocommerce-marketplace__product-loader-cards">
-				<div className="woocommerce-marketplace__product-loader-divider divider-1"></div>
-				<div className="woocommerce-marketplace__product-loader-divider divider-2"></div>
-				<div className="woocommerce-marketplace__product-loader-divider divider-3"></div>
+		<div className="woocommerce-marketplace__product-list">
+			{ hasTitle !== false &&
+				<ProductListHeader title="" groupURL={ null } />
+			}
+			<div className="woocommerce-marketplace__product-list-content">
+				{[...Array(placeholderCount)].map(() =>	<ProductCard />	)}
```

Comment:
Linting errors suggested below. Also seeing the missing key JSX error here too.

```suggestion
			{ hasTitle !== false && (
				<ProductListHeader title="" groupURL={ null } />
			) }
			<div className="woocommerce-marketplace__product-list-content">
				{ [ ...Array( placeholderCount ) ].map( () => (
					<ProductCard />
				) ) }
```